code/client.dm

```diff
@@ -677,6 +679,12 @@ var/global/list/vpn_ip_checks = list() //assoc list of ip = true or ip = false.
 	// Set view tint
 	view_tint = winget( src, "menu.set_tint", "is-checked" ) == "true"
 
+	// Set parallax
+	parallax = winget( src, "menu.set_parallax", "is-checked" ) == "true"
+
+	if(mob?.parallax && !parallax)
```

Comment:
why is mob null

Reply:
i completely didnt realize that i didnt need to use .? there